index.html

```diff
@@ -7,9 +7,23 @@
     <link href="./index.css" rel="stylesheet" />
   </head>
   <body>
-    <nav></nav>
-    <aside></aside>
-    <main></main>
+    <nav class="horizontal-padding">
+      <img id="logo" src="./assets/logo.png" />
+      <div class="nav-links">
+        <img class="nav-option" src="./assets/bell.svg" />
+        <img class="nav-option" src="./assets/settings.svg" />
+        <div class="select-container">
+          <img class="profile-image" src="assets/dp.jpg" />
+          <select id="drop-down-options" name="otions">
+            <option value="Edit Profile">Sara John</option>
+            <option value="Edit Profile">Edit Profile</option>

```

Comment:
Option value should not contain space, it should be small case and separated by a hyphen. In this case, `username`, `edit-profile`, `change-password`, and `log-out`.